### DISCUSSION TITLE
Can I make the layout minimal for neovim?

### DISCUSSION THREAD

**Discussion by ayushnix** [sayanarijit/xplr#440] [Q&A] [answered]
Hey, I was searching for something similar to [nvim-tree](https://github.com/kyazdani42/nvim-tree.lua) when I remembered that I had xplr installed because of [gpg-tui](https://github.com/orhun/gpg-tui). I thought maybe I could use this with neovim?

I've managed to strip the UI down to just the table using this in `~/.config/xplr/init.lua`

```lua
version = '0.17.2'
local xplr = xplr

xplr.config.layouts.custom.nvim = 'Table'

local general = {
  show_hidden = true,
  enable_recover_mode = false,
  hide_remaps_in_help_menu = true,
  initial_layout = 'nvim',
  initial_mode = 'default',
}

for k, v in pairs(general) do
  xplr.config.general[k] = v
end
```

but the permissions, size, and type columns are still active as show in this screenshot.

![2022-02-09_01-49](https://user-images.githubusercontent.com/79408161/153069440-456943d8-b183-4e98-8fd8-ea28f2e95fe8.png)

Can I hide/disable them using the config?

**Comment by sayanarijit** [comment, marked as answer]
Yes... See this: https://github.com/sayanarijit/zentable.xplr/blob/main/init.lua

**Reply by sayanarijit** [reply]
Also, I use https://github.com/sayanarijit/xplr.vim with neovim.

**Comment by ayushnix** [comment]
Thanks!
